trento/adoc/trento-mcp-install.adoc

```diff
@@ -0,0 +1,22 @@
+include::product-attributes.adoc[]
+
+[[sec-trento-mcp-install]]
+:revdate: 2025-11-17
+=== Installing {tr_mcp}
+
+The {tr_mcp} can be deployed in several ways depending on your infrastructure and requirements.
```

Comment:
There are just two ways to deployed Trento server. Let's change "several" with "different":
```suggestion
The {tr_mcp} can be deployed in different ways depending on your infrastructure and requirements.
```